arcade/paths.py

```diff
@@ -71,6 +105,15 @@ def __init__(self, barriers: Union[List, Tuple, Set],
             self.movement_directions = (1, 0), (-1, 0), (0, 1), (0, -1)  # type: ignore
 
     def get_vertex_neighbours(self, pos: Point) -> List[Tuple[float, float]]:
+        """
+        Returns vertexes around the point
```

Comment:
```suggestion
Return neighbors for this point according to ``self.movement_directions``

These are not guaranteed to be reachable or valid points.
```
The implementation does not seem to guarantee the points are reachable, only that they match the accepted movement directions for this grid.